src/SharedResources/components/Alert.ts

```diff
@@ -0,0 +1,15 @@
+import { SFC, createElement } from "react";
+import * as classNames from "classnames";
+
+export interface AlertProps {
+    bootstrapStyle?: "default" | "primary" | "success" | "info" | "warning" | "danger";
+    className?: string;
+}
+
+export const Alert: SFC<AlertProps> = ({ bootstrapStyle, className, children }) =>
```

Comment:
Please note, `SFC` was deprecated in the new versions of React. Instead of that, use `FunctionComponent`

Reply:
Thanx. Fixed